corehq/apps/app_manager/templates/app_manager/partials/case_list_setting.html

```diff
@@ -21,10 +21,14 @@
             $(function () {
                 function updateCaseListLabel() {
                     $(this).next()[$(this).val() === 'true' ? 'show' : 'hide']();
+                    $(this).parent().parent().next()[$(this).val() === 'true' ? 'show' : 'hide']();
```

Comment:
lol that is pretty brutal. i'm not quite sure what you're selecting, though i think it's the  slug media. could you do this:
```
$('#{{ SLUG }}-media')[$(this).val() === 'true' ? 'show' : 'hide']()
```
also if you change the value of the select to 'show' and 'hide', you can simplify further to:
```
$('#{{ SLUG }}-media')[$(this).val()]()
```